Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp

```diff
@@ -314,3 +324,30 @@ SpectralFieldData::BackwardTransform( MultiFab& mf,
         }
     }
 }
+
+#ifdef AMREX_USE_GPU
+std::string
+SpectralFieldData::cufftErrorToString (const cufftResult& err)
+{
+    const auto res2string = std::map<cufftResult, std::string>{
```

Comment:
Can potentially be an unordered map: https://en.cppreference.com/w/cpp/container/unordered_map